projects/ngx-mask-lib/src/lib/ngx-mask.service.ts

```diff
@@ -184,6 +183,12 @@ export class NgxMaskService extends NgxMaskApplierService {
         ) {
             this.decimalMarker = MaskExpression.COMMA;
         }
+        if (
+            this.thousandSeparator === MaskExpression.COMMA &&
+            this.decimalMarker === MaskExpression.COMMA
```

Comment:
lets discuss this case ? The main question could be thousandSeparator and decimalMarker the same ?